docs/containers.conf.5.md

```diff
@@ -710,11 +710,11 @@ and pods are visible.
 
 **network_cmd_path**=""
 
-Path to the slirp4netns binary.
+Path to the slirp4netns or pasta binary.
```

Comment:
This is not used for pasta

Reply:
Why not?  How do I specify an alternate pasta path?  I think we need to change the name of these options to make it easier for future users.

Reply:
helper_binaries_dir is used. It does not scale very well to add a new option for all our binaries.
And at this point renaming the options is a no-go for me. They are used by existing users so there is no point in breaking them. If you think the docs are lacking we can explain this better in the doc text.

Reply:
Then this option should be removed/hidden and only use helper_binaries_dir for finding slirp4netns.  This is our chance to clean up prior mistakes.  If I see network_options and network_cmd_path I am not going to read  a lot of text about which network they support and if I do, I will think less of the tool for being disorganized.